### PR TITLE
Monthly services

### PR DESCRIPTION
**Pull request by hadiaHani**
complete [EditProfile / change password /add service to resident/edit resident view/expense details view/add service]

### CODE REVIEW

**Review by Muhanned-Anwar — APPROVED** [review]

**Review by DevHeyamAwad — APPROVED** [review]

**Review by Muhanned-Anwar — APPROVED** [review]

**Review by DevHeyamAwad — APPROVED** [review]

**Review by DevHeyamAwad — APPROVED** [review]

**Review by Muhanned-Anwar — APPROVED** [review]

File: lib/features/monthly_services/presentation/view/services_view.dart [hunk 1]
```diff
@@ -48,7 +51,7 @@ class ServicesScreen extends StatelessWidget {
                         crossAxisAlignment: CrossAxisAlignment.center,
                         children: [
                           Text(
-                            "${ManagerStrings.type}${controller.monthlyServicesDetailesList[index].type.toString() == "services" ? ManagerStrings.monthleyServicesType : " "}",
+                            "${ManagerStrings.type}${monthlyServicesDetailesList[index].type.toString() == "services" ? ManagerStrings.monthleyServicesType : " "}",
```
Comment (Muhanned-Anwar) [inline lib/features/monthly_services/presentation/view/services_view.dart:54]: create function in controller for this logic

File: lib/features/monthly_services/presentation/view/services_view.dart [hunk 2]
```diff
@@ -110,7 +110,7 @@ class ServicesScreen extends StatelessWidget {
                           height: ManagerHeight.h14,
                         ),
                         Text(
-                            "${ManagerStrings.required} ${(double.parse(controller.monthlyServicesDetailesList[index].remain!) + double.parse(controller.monthlyServicesDetailesList[index].paid!)).toStringAsFixed(1)}",
+                            "${ManagerStrings.required} ${(double.parse(monthlyServicesDetailesList[index].remain!) + double.parse(monthlyServicesDetailesList[index].paid!)).toStringAsFixed(1)}",
```
Comment (Muhanned-Anwar) [inline lib/features/monthly_services/presentation/view/services_view.dart:113]: move logic to controller in function then import function here

File: lib/features/monthly_services/presentation/view/services_view.dart [hunk 3]
```diff
@@ -127,7 +127,7 @@ class ServicesScreen extends StatelessWidget {
                           height: ManagerHeight.h14,
                         ),
                         Text(
-                            "${ManagerStrings.collector} ${(double.parse(controller.monthlyServicesDetailesList[index].paid!))}",
+                            "${ManagerStrings.collector} ${(double.parse(monthlyServicesDetailesList[index].paid!))}",
```
Comment (Muhanned-Anwar) [inline lib/features/monthly_services/presentation/view/services_view.dart:130]: move to controller as before

File: lib/features/monthly_services/presentation/view/services_view.dart [hunk 4]
```diff
@@ -144,13 +144,13 @@ class ServicesScreen extends StatelessWidget {
                           height: ManagerHeight.h14,
                         ),
                         Text(
-                            "${ManagerStrings.residual} ${(double.parse(controller.monthlyServicesDetailesList[index].remain!))}",
+                            "${ManagerStrings.residual} ${(double.parse(monthlyServicesDetailesList[index].remain!))}",
```
Comment (Muhanned-Anwar) [inline lib/features/monthly_services/presentation/view/services_view.dart:147]: move to controller

File: lib/main.dart [hunk 5]
```diff
@@ -27,7 +27,7 @@ class MyApp extends StatelessWidget {
       minTextAdapt: true,
       builder: (context, child) => GetMaterialApp(
         supportedLocales: const [
-          Locale('ar', 'SA'), // English, no country code
+          Locale('ar', 'SA'),
```
Comment (Muhanned-Anwar) [inline lib/main.dart:30]: move locale as constants

**Review by Muhanned-Anwar — CHANGES_REQUESTED** [review]

**Review by Muhanned-Anwar — APPROVED** [review]
Good Job 👍

**Review by Muhanned-Anwar — APPROVED** [review]

**Pull request merged**